ui/src/app/utils/access-utils.tsx

```diff
@@ -300,8 +300,8 @@ export const getAccessModuleConfig = (
   );
 };
 
-// the modules subject to Annual Access Renewal (AAR), in the order shown on the AAR page.
-export const accessRenewalModules = [
+// the modules subject to Registered Tier Annual Access Renewal (AAR), in the order shown on the AAR page.
+export const rtAccessRenewalModules = [
```

Comment:
lots of opportunities for refactoring here, like deriving this list from module config.  But today I'm going for expediency, to get this into the next release.